src/main/java/friendlybot/Storage.java

```diff
@@ -0,0 +1,110 @@
+package friendlybot;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import friendlybot.task.Deadline;
+import friendlybot.task.Event;
+import friendlybot.task.Task;
+import friendlybot.task.ToDo;
+
+/**
+ * Storage deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private String filePath;
+    private String filePathWithoutFile;
+
+    /**
+     * A constructor for Storage.
+     *
+     * @param filePath Path of the TaskList file stored locally.
+     * @param filePathWithoutFile Path of the TaskList file stored locally, excluding the file.
+     *                            (Used to create relevant directories if they do not exist.)
+     */
+    public Storage(String filePath, String filePathWithoutFile) {
+        this.filePath = filePath;
+        this.filePathWithoutFile = filePathWithoutFile;
+    }
+
+    /**
+     * Loads the text file from the file path, and returns an ArrayList of Task corresponding to the text file.
+     * If the path is not recognised, create the relevant directories along the path.
+     * If the file does not exist, create the file and return an empty ArrayList.
+     *
+     * @return An ArrayList of Tasks in the specified file path.
+     */
+    public ArrayList<Task> load() {
```

Comment:
perhaps the naming for this method could be slightly clearer? and similar to your other method in this class e.g. loadFiles()